### ISSUE TITLE
Error Loading Dependencies

### ISSUE THREAD

**Issue by SterlingButters** [plotly/dash-daq#32]
Greetings, I am currently getting some strange results with `dash-daq`. In the minimum example, I get the `Error Loading Dependencies` message (and it seems to occur regardless of daq component [I didn't try them all]):
** Not deployed on Heroku but tried this anyway: https://github.com/plotly/dash-daq/issues/25
```
import dash
import dash_html_components as html
import dash_daq

app = dash.Dash(__name__)

app.layout = html.Div([
    dash_daq.BooleanSwitch(id='toggle-drag', on=True),
])

if __name__ == '__main__':
    app.run_server(debug=True)
```

The js console message:
```
[Error] TypeError: undefined is not an object (evaluating 'n.subscribe')
componentWillMount — bundle.js:2:716
performInitialMount — react-dom@15.4.2.min.js:1998
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
mountChildren — react-dom@15.4.2.min.js:4034
_createInitialChildren — react-dom@15.4.2.min.js:2507
mountComponent — react-dom@15.4.2.min.js:2440
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
performInitialMount — react-dom@15.4.2.min.js:2003
mountComponent — react-dom@15.4.2.min.js:1976:149
mountComponent — react-dom@15.4.2.min.js:4234
updateChildren — react-dom@15.4.2.min.js:1869
_reconcilerUpdateChildren — react-dom@15.4.2.min.js:4023
_updateChildren — react-dom@15.4.2.min.js:4062
updateChildren — react-dom@15.4.2.min.js:4056
_updateDOMChildren — react-dom@15.4.2.min.js:2587:263
updateComponent — react-dom@15.4.2.min.js:2531:105
receiveComponent — react-dom@15.4.2.min.js:2513
receiveComponent — react-dom@15.4.2.min.js:4247
_updateRenderedComponent — react-dom@15.4.2.min.js:2097
_performComponentUpdate — react-dom@15.4.2.min.js:2089:250
updateComponent — react-dom@15.4.2.min.js:2067:294
receiveComponent — react-dom@15.4.2.min.js:2051
receiveComponent — react-dom@15.4.2.min.js:4247
_updateRenderedComponent — react-dom@15.4.2.min.js:2097
_performComponentUpdate — react-dom@15.4.2.min.js:2089:250
updateComponent — react-dom@15.4.2.min.js:2067:294
performUpdateIfNecessary — react-dom@15.4.2.min.js:2054:211
performUpdateIfNecessary — react-dom@15.4.2.min.js:4251:86
s — react-dom@15.4.2.min.js:4446
perform — react-dom@15.4.2.min.js:5411:103
perform — react-dom@15.4.2.min.js:5411:103
T — react-dom@15.4.2.min.js:4501
close — react-dom@15.4.2.min.js:4474:115
closeAll — react-dom@15.4.2.min.js:5446
perform — react-dom@15.4.2.min.js:5419
T — react-dom@15.4.2.min.js:4501
close — react-dom@15.4.2.min.js:4474:115
closeAll — react-dom@15.4.2.min.js:5446
perform — react-dom@15.4.2.min.js:5419
T — react-dom@15.4.2.min.js:4501
T
closeAll — react-dom@15.4.2.min.js:5446
perform — react-dom@15.4.2.min.js:5419
u — react-dom@15.4.2.min.js:4452:162
r — react-dom@15.4.2.min.js:4358
enqueueSetState — react-dom@15.4.2.min.js:4400
setState — react@15.4.2.min.js:12:7784
handleChange — zipWith.js:19
handleChange
dispatch — core.js:97
(anonymous function) — fetch.js:123
promiseReactionJob
(anonymous function) (dash_renderer.dev.js:33621)
promiseReactionJob
```

Python env:
```
(venv) Sterlings-MacBook-Pro:dash-draggable sterlingbutters$ /Users/sterlingbutters/odrive/AmazonDrive/PycharmProjects/DashComponents/dash-draggable/venv/bin/python --version
Python 3.6.8 :: Anaconda custom (64-bit)

(venv) Sterlings-MacBook-Pro:dash-draggable sterlingbutters$ pip show dash-daq
Name: dash-daq
Version: 0.1.2
Summary: DAQ components for Dash
Home-page: UNKNOWN
Author: The Plotly Team
Author-email: UNKNOWN
License: UNKNOWN
Location: /Users/sterlingbutters/odrive/AmazonDrive/PycharmProjects/DashComponents/dash-draggable/venv/lib/python3.6/site-packages
Requires: 
Required-by: 

```

This error is truly odd because several days ago it appeared to be working.

**Comment by shammamah** [comment]
@SterlingButters Thanks for submitting this -- we're getting issues with the NPM package. I would recommend downgrading to `0.1.1` (https://github.com/plotly/dash-daq/issues/25#issuecomment-464796147). Sorry for the inconvenience.

**Comment by SterlingButters** [comment]
Thank God, I thought it was me. Thanks for getting back - good luck and looking forward to the fix

**Comment by shammamah** [comment]
@SterlingButters Just released a new version `0.1.4` with a fix for this (https://github.com/plotly/dash-daq/issues/25#issuecomment-464832596).

**Comment by SterlingButters** [comment]
Thanks! Ill check it out
